tests/composite_pk/models/tenant.py

```diff
@@ -60,3 +60,12 @@ class TimeStamped(models.Model):
     id = models.SmallIntegerField(unique=True)
     created = models.DateTimeField(auto_now_add=True)
     text = models.TextField(default="", blank=True)
+
+
+class AutoId(models.Model):
+    pk = models.CompositePrimaryKey("id", "name")
+    id = models.BigAutoField()
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        required_db_features = ["supports_autofields_in_composite_pk"]
```

Comment:
Does someone know how to skip checks when `required_db_features` is set?  My tests were failing because it was still failing the check when run with SQLite.